qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -108,7 +104,7 @@ def format(self) -> str:
         Returns:
              The job queue information report.
         """
-        status = api_status_to_job_status(ApiJobStatus(self._status)).value \
+        status = api_status_to_job_status(self._status).value \
```

Comment:
Nit: Only for consistency. The `.value` here could be set to `.name` or the `.name` in `line 81` could be set to `.value`.

Reply:
This is actually done on purpose. A `__repr__` is the representation of the instance, hence something like `status=RUNNING` seems more suited than `status=job is actively running`. The `format()` method, on the other hand, gives a user friendly formatted output, so it uses `status: job is actively running`.